Examples/Swift/ViewController.swift

```diff
@@ -101,7 +101,9 @@ class ViewController: UIViewController, MGLMapViewDelegate {
             self.startMultipleWaypoints()
         }))
         alertController.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: nil))
-        present(alertController, animated: true, completion: nil)
+        DispatchQueue.main.async {
```

Comment:
The entire screen is locked up actually when choosing between the different views. Even when selecting cancel, there is a long wait.